### PR TITLE
Removed Duplicate and Fixed HTML5 Issue

### PR DESCRIPTION
**Pull request by theflametrooper**
Can you please squash my commits because i'm editing this in the browser... @Rafase282

### CODE REVIEW

File: Campsites.md [hunk 1]
```diff
@@ -6,11 +6,11 @@ Whether you live in a major metropolitan area, or a small town, we encourage you
 
 Here are some helpful links:
 
-[Complete Alphabetical List of Campsites](List-of-Free-Code-Camp-city-based-Campsites)
+[Complete Alphabetical List of Campsites](List-of-Free-Code-Camp-city-based-Campsites.md)
```
Comment (abhisekp) [inline Campsites.md]: No need to append the extension `.md`

Comment (theflametrooper) [inline Campsites.md]: It doesn't work, It cannot find the page because there are no file extensions

**Comment by abhisekp** [comment]
### **Note to Giant Squasher**
Please **keep the commit** with commit message `Removed Duplicate Article` as a separate commit. And squash all other commits.

**Comment by Rafase282** [comment]
@abhisekp this is what it would be if I squash:

* Updated to HTML5 em tag

* Removed Duplicate Article

* Fixed Links on Campsites.md

* Removed my mistakes

It will also be one commit. If I do it the old way I would have to create a new PR on the repo with the squashing the way you ask and have it automatically close this one.

**Comment by abhisekp** [comment]
@Rafase282 Use diff tools and see that there is absolutely no change in the last 2 commits. Last is simply a revert of the last of one.

Keep only the first two i.e.
- Updated to HTML5 em tag
- Removed Dupe Article

the last two can be skipped *happily* while rebasing.

And yes, you've to do in the old style. :dancer: :wink:

**Comment by Rafase282** [comment]
@theflametrooper I think you deleted the branch with the changes. I would suggest you open two new PR, one for the html fix and the other for removing the duplicate file. Try to keep unrelated changes separated.

**Comment by theflametrooper** [comment]
@Rafase282 Sure

**Pull request closed without merging**